var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -59,7 +60,7 @@ class Geant4(CMakePackage):
     variant("x11", default=False, description="Optional X11 support")
     variant("motif", default=False, description="Optional motif support")
     variant("qt", default=False, description="Enable Qt support")
-    variant("python", default=False, when="@10.6.2:", description="Enable Python bindings")
+    variant("python", default=False, description="Enable Python bindings", when="@10.6.2:10.7")
```

Comment:
I thought you said python worked for 11.0?

Reply:
Yes, sorry, I'd made a typo! Fixed in extra commit.